plugins/catalog/src/components/CatalogPage/CatalogPage.tsx

```diff
@@ -30,14 +31,13 @@ import React, { useCallback, useMemo, useState } from 'react';
 import { Link as RouterLink } from 'react-router-dom';
 import { EntityFilterGroupsProvider, useFilteredEntities } from '../../filter';
 import { useStarredEntities } from '../../hooks/useStarredEntities';
-import { catalogApiRef } from '../../plugin';
 import { ButtonGroup, CatalogFilter } from '../CatalogFilter/CatalogFilter';
 import { CatalogTable } from '../CatalogTable/CatalogTable';
+import { isOwnerOf } from '../isOwnerOf';
```

Comment:
yeah this could be a candidate to move later as well, if others find use of it ... but maybe it takes a bit of refactoring (like possibly making it async, since if the ownership RFC goes through it may want to do some lookups?)

Reply:
Yes it is! @dhenneke already had this one on the list in #3655, but this PR doesn't cover anything from there yet.

Reply:
I'm about to prepare a follow up PR and I'm already moving it there 😆